docs/readmes/howtos/call_tracing.md

```diff
@@ -0,0 +1,65 @@
+---
+id: call_tracing 
+title: Call Tracing
+hide_title: true
+---
+
+*Last Updated: 1/20/2021*
+
+# Call Tracing
+
+Magma supports basic traffic capture for troubleshooting purposes.
+Monitoring of control messaging flow and other traffic between the Magma access
+gateway and eNodeB devices is possible with this feature.
+
+As of the time of writing, we are in the process of adding filtering for
+specific protocols and allowing custom options through tshark.
+
```

Comment:
Can you add the max size of call tracing buffer we currently support and any other assumptions here.